docs/composition.ipynb

```diff
@@ -0,0 +1,739 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<i>naive plan: (run <b>LazyFrame.explain(optimized=True)</b> to see the optimized plan)</i>\n",
+       "    <p></p>\n",
+       "    <div>AGGREGATE<p></p>\t[[(col(\"HWUSUAL\").fill_null([40.0]).clip([1, 98]).sum()./Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/opendp/lib/opendp.abi3.so:noise()) / (len())].alias(\"Mean Hours Worked\"), col(\"HWUSUAL\").fill_null([40.0])./Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/opendp/lib/opendp.abi3.so:discrete_quantile_score()./Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/opendp/lib/opendp.abi3.so:report_noisy_max_gumbel()./Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/opendp/lib/opendp.abi3.so:index_candidates().alias(\"Median Hours Worked\")] BY [col(\"ILOSTAT\")] FROM<p></p>  DF [\"COEFF\", \"QUARTER\", \"REFYEAR\", \"REFWEEK\"]; PROJECT */77 COLUMNS; SELECTION: \"None\"</div>"
+      ],
+      "text/plain": [
+       "<wrap [3 cols, {\"ILOSTAT\": Int64 … \"Median Hours Worked\": Int64}] at 0x16D8417C0>"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#define polars queries\n",
+    "proposed_plan = context.query().groupby(\"ILOSTAT\").agg(\n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.mean((1,98), scale = 8).alias(\"Mean Hours Worked\"), \n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.median(candidates, 8).alias(\"Median Hours Worked\"),\n",
+    ")\n",
+    "proposed_plan"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we set up a measurement to execute the sequential creaties and pass in our `df_domain` and `proposed_plan`. We can pass our data to this measurement and the collect our results. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "shape: (1, 3)\n",
+      "┌─────────┬───────────────────┬─────────────────────┐\n",
+      "│ ILOSTAT ┆ Mean Hours Worked ┆ Median Hours Worked │\n",
+      "│ ---     ┆ ---               ┆ ---                 │\n",
+      "│ i64     ┆ f64               ┆ i64                 │\n",
+      "╞═════════╪═══════════════════╪═════════════════════╡\n",
+      "│ 1       ┆ 37.671921         ┆ 29                  │\n",
+      "└─────────┴───────────────────┴─────────────────────┘\n",
+      "<opendp.polars.OnceFrame object at 0x12ea6f740>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# IN SERVER\n",
```

Comment:
Are we shifting from explorative code running locally to code that could run in the secure environment?

Reply:
don't think so